docs/docs/conceptual/rendering-options.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Rendering Options
+---
+
+Gatsby is historically known as a static site generator enhanced with [React Hydration][1].
+But starting with Gatsby 4, you can choose alternative rendering modes in addition to static site generation (SSG).
+Now Gatsby covers all possible use-cases you may face as a web developer.
+
+### What is a rendering mode?
+
+Rendering mode defines at what stage the user-facing HTML is generated. It can happen at build time
+(SSG or pre-rendering), during HTTP request (server-side rendering) or locally in the browser
+with Javascript (client-side rendering).
+
+> For an in-depth explanation of each of those approaches and their trade-offs we
+> highly recommend the ["Rendering on the Web"][3] article from the Chrome team.
+
+As already mentioned, Gatsby always supported SSG and [client-side rendering][4], and now the two
+other rendering options are available: Deferred Static Generation (DSG) and Server-side rendering (SSR).
+Let's dive into what each of those modes mean.
+
+## Static Site Generation (SSG)
+
+SSG is the default rendering mode in Gatsby. Gatsby generates all assets and HTML for all pages at build time on a
```

Comment:
this probably need better wording than my suggestion, but "for all pages" is the "old" way, and now it "doesn't have to be all pages at build time"

```suggestion
SSG is the default rendering mode in Gatsby. Gatsby generates all assets and HTML for all SSG pages at build time on a
```